global/iam/users/main.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = ">=0.12"
```

Comment:
**Question:** Does this have to be in sync with all the `main.tf` files? If that's the case, is there a way to put it as a variable?

Reply:
doesn't have to be in the main.tf file, it can be in any .tf file in the directory, but it is a good practice put it on the main.tf file.

This is the terraform configuration block https://www.terraform.io/docs/configuration/terraform.html, you can't use variables here unfortunately